src/content/docs/en/guides/deploy/netlify.mdx

```diff
@@ -64,6 +64,8 @@ If you prefer to install the adapter manually instead, complete the following tw
         }),
       });
     ```
+
+    If you use Middleware to implement authentication, redirects, or similar things, you may need to enable this flag. Otherwise, your Middleware will not run at runtime for pre-rendered pages.
```

Comment:
Same here re: capitalisation, but also suggesting flipping the wording for a more positive “do X to achieve Y” pattern rather than “do X to avoid Z”.

```suggestion
    
    Set `edgeMiddleware: true` if you want to run middleware for pre-rendered pages. This allows you to use middleware to implement authentication, redirects, or similar things, while still using static HTML output.
```

Reply:
Great call, Chris!

For your consideration, I'll also throw this slight tweak out there to frontload the "to do X" part:

> To run middleware for pre-rendered pages, set `edgeMiddleware: true`. ...

Reply:
Ah, great call, yes. I honestly initially aimed for that and didn’t quite land on a formulation I liked, but this works great. Will update my suggestion to match!